drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
-	callback    driverapi.DriverCallback
```

Comment:
Thats exactly the question. How is it handled ?
Just by removing it from here and not giving an alternative just doesn't show the design intent. Have you thought of a design for that ? This is integral part of the remote driver design and hence am curious to see your thought process.